src/plugins/saved_search/public/services/saved_searches/saved_searches_utils.test.ts

```diff
@@ -91,28 +89,6 @@ describe('saved_searches_utils', () => {
     });
   });
 
-  describe('throwErrorOnSavedSearchUrlConflict', () => {
```

Comment:
I like that this logic lives in `getSavedSearch` now, but it would be great if we moved this test or a similar one with it.